docs/topics/scale.md

```diff
@@ -38,6 +38,6 @@ This command will re-use the `apimodel.json` file inside the output directory as
 |--certificate-path|depends| The path to the file which contains the client certificate. This is required if the auth-method is set to client_certificate|
 |--node-pool|depends|Required if there is more than one node pool. Which node pool should be scaled.|
 |--new-node-count|yes|Desired number of nodes in the node pool.|
-|--master-FQDN|depends|When scaling down a kubernetes cluster this is required. The master FDQN so that the nodes can be cordoned and drained before removal. This should be output as part of the create template or it can be found by looking at the public ip addresses in the resource group.|
+|--apiserver|when scaling "in"|apiserver endpoint, needed to cordon/drain VMs. This should be output as part of the create template or it can be found by looking at the public ip addresses in the resource group.|
```

Comment:
I'm not sure scale "in" is better here. Most references in the k8s world and Kubernetes itself use `scale up` and `scale down`:

https://docs.microsoft.com/en-us/azure/aks/scale-cluster
https://cloud.google.com/kubernetes-engine/docs/concepts/cluster-autoscaler
https://kubernetes.io/docs/concepts/workloads/controllers/deployment/

Reply:
Sadly this is correct